app/secure/models/PaymentRequest.class.js

```diff
@@ -0,0 +1,15 @@
+'use strict'
```

Comment:
This I would imagine is a common class so should probable be in `common/classes`

Reply:
Coolio 👍